src/database/common.c

```diff
@@ -85,27 +83,25 @@ int dbquery(sqlite3* db, const char *format, ...)
 
 	if(query == NULL)
 	{
-		logg("Memory allocation failed in dbquery()");
+		log_err("Memory allocation failed in dbquery()");
 		return SQLITE_ERROR;
 	}
 
 	// Log generated SQL string when dbquery() is called
 	// although the database connection is not available
 	if(db == NULL)
 	{
-		logg("dbquery(\"%s\") called but database is not available!", query);
+		log_err("dbquery(\"%s\") called but database is not available!", query);
 		sqlite3_free(query);
 		return SQLITE_ERROR;
 	}
 
-	if(config.debug & DEBUG_DATABASE)
-	{
-		logg("dbquery: \"%s\"", query);
-	}
+log_debug(DEBUG_DATABASE, "dbquery: \"%s\"", query);
+
 
 	int rc = sqlite3_exec(db, query, NULL, NULL, NULL);
 	if( rc != SQLITE_OK ){
-		logg("ERROR: SQL query \"%s\" failed: %s",
+		log_err("ERROR: SQL query \"%s\" failed: %s",
```

Comment:
This will result in `ERR: ERROR: ...`
```suggestion
		log_err("SQL query \"%s\" failed: %s",
```